90zfsbootmenu/zfsbootmenu-help.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+# vim: softtabstop=2 shiftwidth=2 expandtab
+WIDTH="$( tput cols )"
+PREVIEW_SIZE=$(( WIDTH - 26 ))
+
+[ -z "${FUZZYSEL}" ] && FUZZYSEL="fzf"
+
+center() {
+  printf "%*s" $(( (${#1} + WIDTH ) / 2)) "${1}"
+}
+
+colorize() {
+  color="${1}"
+  shift
+  case "${color}" in
+    black) echo -e -n '\033[0;30m' ;;
+    red) echo -e -n '\033[0;31m' ;;
+    green) echo -e -n '\033[0;32m' ;;
+    orange) echo -e -n '\033[0;33m' ;;
+    blue) echo -e -n '\033[0;34m' ;;
+    magenta) echo -e -n '\033[0;35m' ;;
+    cyan) echo -e -n '\033[0;36m' ;;
+    lightgray) echo -e -n '\033[0;37m' ;;
+    darkgray) echo -e -n '\033[1;30m' ;;
+    lightred) echo -e -n '\033[1;31m' ;;
+    lightgreen) echo -e -n '\033[1;32m' ;;
+    yellow) echo -e -n '\033[1;33m' ;;
+    lightblue) echo -e -n '\033[1;34m' ;;
+    lightmagenta) echo -e -n '\033[1;35m' ;;
+    lightcyan) echo -e -n '\033[1;36m' ;;
+    white) echo -e -n '\033[1;37m' ;;
+    *) echo -e -n '\033[0m' ;;
+  esac
+  echo -e -n "$@"
+  echo -e -n '\033[0m'
+}
+
+help_pager() {
+  WANTED="${1}"
+
+  SORTED=()
+  for SECTION in "${SECTIONS[@]}"; do
+    if ! [[ $SECTION =~ ${WANTED} ]]; then
+      SORTED+=("${SECTION}")
+    else
+      FINAL="${SECTION}"
+    fi
+    done
+  SORTED+=("${FINAL}")
+
+  printf '%s\n' "${SORTED[@]}" | ${FUZZYSEL} \
+    --prompt 'Topic >' \
+    --with-nth=2.. \
+    --bind pgup:preview-up,pgdn:preview-down \
+    --preview="$0 -s {1}" \
+    --preview-window="right:${PREVIEW_SIZE}:sharp:wrap" \
+    --tac \
+    --color='border:6'
+}
+
+# shellcheck disable=SC2034
+read -r -d '' MAIN <<EOF
+$( colorize magenta "$( center "Main Menu")" )
+$( colorize lightblue "[ENTER] boot" )
+Boot the selected boot environment, with the listed kernel and kernel command line visible at the top of the screen.
+
+$( colorize lightblue "[ALT+K] kernel" )
+Access a list of kernels available in the boot environment.
+
+$( colorize lightblue "[ALT+D] set bootfs" )
+Set the selected boot environment as the default for the pool.
+
+The operation will gracefully fail if the pool can not be set $( colorize red "read/write" ).
+
+$( colorize lightblue "[ALT+S] snapshots" )
+Access a list of snapshots of the selected boot environment. New boot environments can be created here.
+
+$( colorize lightblue "[ALT+C] cmdline" )
+Temporarily edit the kernel command line that will be used to boot the next kernel and boot environment. This change is not persisted between boots.
+
+$( colorize lightblue "[ALT+P] Pool status" )
```

Comment:
What is the capitalization? Here it's `Pool status`, but above it's `set bootfs`.

Reply:
I was mirroring the descriptions used in the footer. Those should probably get a normalization pass next.